scripts/setup_cameras_v3link.sh

```diff
@@ -0,0 +1,467 @@
+        echo "    name = ov2312"
+        echo "    format = $OV2312_CAM_FMT"
+        echo "    subdev_id = $CAM_SUBDEV_NAME"
+        echo "    isp_required = yes"
+        echo "    ldc_required = no"
+
+        ((i++))
+    done
+    done
+}
+
+setup_imx219(){
+    i=0
+    for media_id in {0..3}; do
+    # UB953 FORMATS
+    UB960_FMT_STR=""
+    CDNS_FMT_STR=""
+    CSI2RX_FMT_STR=""
+    for name in `media-ctl -d $media_id -p | grep entity | grep imx219 | cut -d ' ' -f 5`; do
+
+        CAM_SUBDEV=`media-ctl -d $media_id -p -e "imx219 $name" | grep v4l-subdev | awk '{print $4}'`
+        media-ctl -d $media_id -V "'imx219 $name':0 $IMX219_CAM_FMT"
+
+        UB953_NAME=`media-ctl -d $media_id -p -e "imx219 $name" | grep ub953 | cut -d "\"" -f 2`
+        media-ctl -d $media_id -V "'$UB953_NAME':0 $IMX219_CAM_FMT"
+
+        UB960_NAME=`media-ctl -d $media_id -p -e "$UB953_NAME" | grep ub960 | cut -d "\"" -f 2`
+        UB960_PAD=`media-ctl -d $media_id -p -e "$UB953_NAME" | grep ub960 | cut -d : -f 2 | awk '{print $1}'`
+
+        CSI_BRIDGE_NAME=`media-ctl -d $media_id -p -e "$UB960_NAME" | grep csi-bridge | cut -d "\"" -f 2`
+        CSI2RX_NAME=`media-ctl -d $media_id -p -e "$CSI_BRIDGE_NAME" | grep "ticsi2rx\"" | cut -d "\"" -f 2`
+
+        LAST_PAD=`echo ${ALL_UB960_FMT_STR[$media_id,$UB960_NAME]} | rev | cut -d'/' -f 1 | rev`
+        LAST_PAD=${LAST_PAD:0:1}
+        if [[ "$LAST_PAD" == "" ]] ; then
+            NEXT_PAD=$UB960_PAD
+        else
+            NEXT_PAD=$(($LAST_PAD+1))
+        fi
+        CSI2RX_CONTEXT_NAME="$CSI2RX_NAME context $((NEXT_PAD))"
+
+        UB960_FMT_STR="${UB960_PAD}/0 -> 4/$(($NEXT_PAD)) [1]"
+        CDNS_FMT_STR="0/${NEXT_PAD} -> 1/$(($NEXT_PAD)) [1]"
+        CSI2RX_FMT_STR="0/${NEXT_PAD} -> $(($NEXT_PAD+1))/0 [1]"
+
+        # Append UB960 Routes
+        if [[ -v "ALL_UB960_FMT_STR[$media_id,$UB960_NAME]" ]] ; then
+            ALL_UB960_FMT_STR[$media_id,$UB960_NAME]="${ALL_UB960_FMT_STR[$media_id,$UB960_NAME]}, $UB960_FMT_STR"
+        else
+            ALL_UB960_FMT_STR[$media_id,$UB960_NAME]="$UB960_FMT_STR"
+        fi
+
+        # Append CDNS Routes
+        if [[ -v "ALL_CDNS_FMT_STR[$media_id,$CSI_BRIDGE_NAME]" ]] ; then
+            ALL_CDNS_FMT_STR[$media_id,$CSI_BRIDGE_NAME]="${ALL_CDNS_FMT_STR[$media_id,$CSI_BRIDGE_NAME]}, $CDNS_FMT_STR"
+        else
+            ALL_CDNS_FMT_STR[$media_id,$CSI_BRIDGE_NAME]="$CDNS_FMT_STR"
+        fi
+
+        # Append CSIRX Routes
+        if [[ -v "ALL_CSI2RX_FMT_STR[$media_id,$CSI2RX_NAME]" ]] ; then
+            ALL_CSI2RX_FMT_STR[$media_id,$CSI2RX_NAME]="${ALL_CSI2RX_FMT_STR[$media_id,$CSI2RX_NAME]}, $CSI2RX_FMT_STR"
+        else
+            ALL_CSI2RX_FMT_STR[$media_id,$CSI2RX_NAME]="$CSI2RX_FMT_STR"
+        fi
+        CAM_DEV=`media-ctl -d $media_id -p -e "$CSI2RX_CONTEXT_NAME" | grep video | awk '{print $4}'`
+        CAM_DEV_NAME=/dev/video-imx219-cam$i
+
+        CAM_SUBDEV_NAME=/dev/v4l-imx219-subdev$i
```

Comment:
Hey,

I tested this on AM62A and it seems to work. Just a small suggestion,
can we change these names to "rpi" instead of imx219. So instead of calling it
"/dev/video-imx219-cam0" we call it "/dev/video-rpi-cam0". Same for subdev.

The reason is that original setup_cameras.sh script name it with "rpi" so all our config files and documentation refers to /dev/video-rpi-cam0

Reply:
@abhaychirania2411 , I would think "imx219" makes more sense because there are other "rpi" cameras than just imx219. Are there other scripts that depend on this script and use "rpi" to reference the imx219 camera?

Reply:
@abhaychirania2411 , we may connect different Raspberry Pi compatible cameras to the V3link board, such as imx219, ov5647, etc.

Reply:
If dealing with these queries are fine if they come, we might as well go ahead and change it to imx219 for this release.

Reply:
We should be fine with answering queries from customers. The script prints out the video node name and that should help. I would avoid "rpi" as it is not explicit and Rpi is TI's competitor.

Reply:
Makes sense. I'll merge this PR and update the existing scripts as well to use IMX219. :)